modules/apps/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/frontend/taglib/servlet/taglib/UserPasswordScreenNavigationEntry.java

```diff
@@ -55,8 +58,15 @@ public boolean isVisible(User user, User selUser) {
 		if (selUser == null) {
 			return false;
 		}
+		try {
+			return selUser.getPasswordPolicy().isChangeable();
```

Comment:
The acceptance criteria mentioned `We need to pay attention for having the LDAP or any SSO configured.`, so i'm curious how if the current user is controlled by an LDAP Password Policy?

Reply:
@hhtank The requirement from Hong Sheng was to hide the password according to Liferay's password policy. Since this controlled by the Admin we don't care about SSO.